### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joliyea**
- Improve the explanation of how to create a new blog post.
- Update structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,86 +9,112 @@ Solltest du Hilfe brauchen oder eine Frage haben, mach gerne ein [Issue](https:/
...
+git push
+```
+7. Pull request eröffnen: Hier auf GitHub sollte dazu ganz oben ein Hinweis auftauchen. Einfach den Anweisungen folgen.
+8. Fertig!
 
-Code: [MIT](./LICENSE). Inhalt: [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/deed.de).
 
-## Die Struktur im Überblick
+## Hinweise
+* Den `main`-Branch, aus dem sich die **Seite automatisch baut**, kann nicht direkt verändert werden. Stattdessen bringe deine Anderungen in einen eigenen Branch und stelle einen Pull-Request. Pull-Requests erhalten dank netlify automatisch eine Vorschau-URL, du findest diese dann als Preview in den Checks deines PRs.
```
Comment (ubergesundheit) [inline README.md:47]:
```suggestion
* Den `main`-Branch, aus dem sich die **Seite automatisch baut**, kann nicht direkt verändert werden. Stattdessen bringe deine Änderungen in einen eigenen Branch und stelle einen Pull-Request. Pull-Requests erhalten dank netlify automatisch eine Vorschau-URL, du findest diese dann als Preview in den Checks deines PRs.
```

File: README.md [hunk 2]
```diff
@@ -9,86 +9,112 @@ Solltest du Hilfe brauchen oder eine Frage haben, mach gerne ein [Issue](https:/
...
+* Am besten bearbeitest du die Seite **lokal und testest** dann einmal mit [`hugo -D`*](#Gebaut-mit-Hugo), ob sie wirklich funktioniert. Dadurch hilfst du, Fehler und damit auch zusätzliche Arbeit zu vermeiden.
```
Comment (ubergesundheit) [inline README.md:48]: Der Link geht nicht, das Sternchen versteh ich nicht.

```suggestion
* Am besten bearbeitest du die Seite **lokal und testest** dann einmal mit `hugo -D`, ob sie wirklich funktioniert. Dadurch hilfst du, Fehler und damit auch zusätzliche Arbeit zu vermeiden.
```

**Review by ubergesundheit — APPROVED** [review]
LGTM

Although the instructions here already require a general understanding of git etc..

A nice alternative would be to describe the contribution process without requiring to clone the project and do everything using the GitHub UI

**Comment by joliyea** [comment]
* Added changes.
* Also added necessity of a description "how to use the GitHub UI to add blog posts" to local list "manmuesstemal". 😊

**Pull request merged**